kvbc/CMakeLists.txt

```diff
@@ -19,7 +19,6 @@ add_library(kvbc  src/ClientImp.cpp
     src/direct_kv_storage_factory.cpp
     src/merkle_tree_storage_factory.cpp
     src/pruning_handler.cpp
-    src/pruning_reserved_pages_client.cpp
```

Comment:
This PR removes adaptive pruning, is this file also relevant for the concurrent pruning?

Reply:
PR is also going to remove concurrent pruning code. I've updated title & description of PR.